File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoPullRequestsFragmentComponent.java
package mb00.android.codehub.ui.repo.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.repo.view.RepoPullRequestsFragment;


@Subcomponent(modules = RepoPullRequestsFragmentModule.class)
public interface RepoPullRequestsFragmentComponent extends AndroidInjector<RepoPullRequestsFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<RepoPullRequestsFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoPullRequestsFragmentComponent.kt
package mb00.android.codehub.ui.repo.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.repo.view.RepoPullRequestsFragment


@Subcomponent(modules = [
    RepoPullRequestsFragmentModule::class
])
interface RepoPullRequestsFragmentComponent : AndroidInjector<RepoPullRequestsFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<RepoPullRequestsFragment>()

}
